src/components/Map/MapView.tsx

```diff
@@ -107,6 +108,7 @@ export const MapView = () => {
             />
           ))}
 
+        <AccordionDemo />
```

Comment:
Have we made an overflow behavior decision about the accordion vs info panes (whatever's already on the left)?

Reply:
resolved: last clicked on is displayed.